.github/workflows/build.yml

```diff
@@ -172,3 +184,57 @@ jobs:
       run: |
         cd build/scripts
         ../Debug/bin/unit_tests
+
+
+
+  build-arm64-docker:
+    name: Build for ARM64 on Docker
```

Comment:
merged the ARM build into this file, for easier integration to PYPI,Release jobs